debsources/app/static/javascript/debsources.js

```diff
@@ -184,3 +184,42 @@ var debsources = {
 		}
 	}
 };
+
+var debsources_responsive = {
+    // debsources_responsive
+    // 
+    // relys on:
+    // - JQuery (as does bootstra)
+    render_bg: function(url) {
+      if (typeof url === typeof "string") {
+        $('#bg-wrapper').css('background-image', 'url('+ url + ')')
+        console.log("Changed background image")
+      }
+    },
+
+    fetch_url: function() {
+      /* Currently uses hacky invisible p to get url
+       * This is due to the fact we use url_for(), a jinja method.
+       * */
+      var url = $("#bg-url").attr('data-url');
+      return url;
+    },
+
+    mobile_infobox: function() {
+      var infobox = $(".pkginfobox");
+      if (infobox) {
+        console.log("Running infobox for mobiles");
+      }
+    },
+
+    run: function() {
+      console.log("Running Responsive Utilities");
+      this.render_bg(this.fetch_url());
+      this.mobile_infobox();
+    }
+};
+
+$(document).ready( function() {
+  console.log("Debsources Javascript Active");
+  debsources_responsive.run();
+});
```

Comment:
I still don't get it. Why not keep the current solution? I agree it's not optimal, but IMO better than javascript!